### DISCUSSION TITLE
0.49.0

### DISCUSSION THREAD

**Discussion by adamghill** [django-commons/django-unicorn#471] [General]
- Fix: Handle inherited (i.e. subclassed) models [#459](https://github.com/adamghill/django-unicorn/issues/459).
- Fix: Support abbreviated `u:view` ([#464](https://github.com/adamghill/django-unicorn/pull/464) by [nerdoc](https://github.com/nerdoc)).
- Add version and build date to minified JavaScript for easier debugging.

<hr /><em>This discussion was created from the release <a href='https://github.com/adamghill/django-unicorn/releases/tag/0.49.0'>0.49.0</a>.</em>

**Comment by garethpenrose** [comment]
Hi there, I am getting an error when moving from 0.48.0 to 0.49.0 

File "/usr/local/lib/python3.7/site-packages/django_unicorn/utils.py", line 6, in <module>
from pprint import pp
ImportError: cannot import name 'pp' from 'pprint' (/usr/local/lib/python3.7/pprint.py)

Any dependency updates I need to make or is this an issue?

**Reply by adamghill** [reply]
Oh shoot, that's definitely a bug I introduced. I'll fix this and release 0.49.1 as soon as possible.

**Reply by adamghill** [reply]
Made a PR: https://github.com/adamghill/django-unicorn/pull/479

**Comment by garethpenrose** [comment]
Ah fair play Adam, I will get the latest once your PR is approved and I will test and confirm asap :)

**Reply by adamghill** [reply]
I have published [0.49.1](https://github.com/adamghill/django-unicorn/releases/tag/0.49.1) with the fix, let me know if you run into any other problems!

**Reply by garethpenrose** [reply]
I've rebuilt and retested and that no longer produces the issue as outlined above. Great work on this project and very much appreciate your swift resolution to the post. Great stuff Adam.
